x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/knowledge_base_edit_manual_entry_flyout.tsx

```diff
@@ -42,20 +43,33 @@ export function KnowledgeBaseEditManualEntryFlyout({
   const { mutateAsync: createEntry, isLoading } = useCreateKnowledgeBaseEntry();
   const { mutateAsync: deleteEntry, isLoading: isDeleting } = useDeleteKnowledgeBaseEntry();
 
+  const isSystemPromptDefaultValue = entry?.labels?.category === 'instruction' ?? false;
+  const [isSystemPrompt, setIsSystemPrompt] = useState(isSystemPromptDefaultValue);
```

Comment:
I think we agreed to rename this to user instruction in the code and UI, just leaving a comment for the same thing.

Reply:
Removed this per our conversation today.